Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -1228,7 +1228,9 @@ def audit_urls
         end
       elsif strategy <= SubversionDownloadStrategy
         next unless DevelopmentTools.subversion_handles_most_https_certificates?
-        unless Utils.svn_remote_exists url
+        if !Utils.svn_available?
+          problem "No valid version of svn found"
```

Comment:
Actually: this shouldn't be a `problem` because it's not an issue with the formula that's being `audit`ed. Instead I'd just do `next unless Utils.svn_available?`

Reply:
Got it! Made the change. Thanks :)